Multisig/UI/UI Library/Tab Views/GNO Tab View/GNOTabButton.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  GNOTabButton.swift
+//  Multisig
+//
+//  Created by Dmitry Bespalov on 19.05.20.
+//  Copyright © 2020 Gnosis Ltd. All rights reserved.
+//
+
+import SwiftUI
+
+struct GNOTabButton<SelectionValue: Hashable, Content: View>: View {
```

Comment:
this is almost identical to the TopTabButton, in fact, it is a modified copy.

I have not had a chance to think  how to do it in a common/generic way to provide different styling.